internal/services/mysql/mysql_server_resource_test.go

```diff
@@ -266,27 +266,32 @@ resource "azurerm_resource_group" "test" {
 }
 
 resource "azurerm_mysql_server" "test" {
-  name                             = "acctestmysqlsvr-%[1]d"
-  location                         = azurerm_resource_group.test.location
-  resource_group_name              = azurerm_resource_group.test.name
-  sku_name                         = "GP_Gen5_2"
-  administrator_login              = "acctestun"
-  administrator_login_password     = "H@Sh1CoR3!"
-  auto_grow_enabled                = true
-  backup_retention_days            = 7
-  create_mode                      = "Default"
-  geo_redundant_backup_enabled     = false
-  ssl_enforcement_enabled          = true
-  ssl_minimal_tls_version_enforced = "TLS1_2"
-  storage_mb                       = 51200
-  version                          = "%[3]s"
+  name                              = "acctestmysqlsvr-%[1]d"
+  location                          = azurerm_resource_group.test.location
+  resource_group_name               = azurerm_resource_group.test.name
+  sku_name                          = "GP_Gen5_2"
+  administrator_login               = "acctestun"
+  administrator_login_password      = "H@Sh1CoR3!"
+  auto_grow_enabled                 = true
+  backup_retention_days             = 7
+  create_mode                       = "Default"
+  geo_redundant_backup_enabled      = false
+  infrastructure_encryption_enabled = false
```

Comment:
this field is ForceNew, so setting this will recreate the Server, which invalidates the Update test - so please split this out to a separate test.